lib/form/paragraph3.flow

```diff
@@ -112,7 +115,14 @@ renderParagraph(
 	widthManager : WigiParagraphWidthManager
 ) -> Form {
 
-	alignment : ParaLineAlignment = extractParaLineAlignment(s);
+	alignment : Maybe<double> = switch (extractParaLineAlignment(s)) {
```

Comment:
@alstrup , looks like we need your advice here
From my point of view, having union for alignment is better that compressing it into some tricky double value.
But @brawaga tells us that this is the common practice in the tropic world. 
If it is common and accepted then we agree to change paragraph to be like tropic. But maybe the change should be opposite - to make tropic be union-based?

Reply:
I did not tell it common in overall Ttropic, I told about TAlign only. Which works also in Material though. Also, this is internal only stuff, so I did not expect this to become a matter of conversation. When I did this, I tried to exclude multiple switches from code of repetitious execution up to paragraph initialization. Now I see maybe better to leave double instead Maybe<double> with exclusive (-1?) value for justify, but I am not sure about what can we get here: it does not seem to cut down if/switch quantity anymore, and I am unaware if we will get  some profit on parameter passing (union vs double).

Reply:
as for me, using distinct data members of union is a way better that tricky double values. it much easy to understand and produce less errors.
even if using double reduce couple of lines of code, it sacrifice understanding what is exactly is happens in the particular line of the code, require to lookup into the definition to understand the meaning of -1, 0, 0.5, 1
I'm not sure it worth it

Reply:
@brawaga ,
I can't see any improvements even on the code level from switching into double stuff. could you explain where it bring real benifits?